Assets/Plugins/Source/Editor/EditorWindows/EOSSettingsWindow.cs

```diff
@@ -148,18 +148,40 @@ protected override async Task AsyncSetup()
             mainEOSConfigFile = await Config.GetAsync<EOSConfig>();
 
             platformSpecificConfigEditors ??= new List<IConfigEditor>();
-            var configEditors = ReflectionUtility.CreateInstancesOfDerivedGenericClasses(typeof(PlatformConfigEditor<>));
             List<string> toolbarStrings = new(new[] { "Main" });
-            foreach (IPlatformConfigEditor editor in configEditors.Cast<IPlatformConfigEditor>())
+            foreach (PlatformManager.Platform platform in Enum.GetValues(typeof(PlatformManager.Platform)))
             {
-                // If the platform for the editor is not available, then do not
-                // display the editor for it.
-                if (!editor.IsPlatformAvailable())
+                // Windows platform overrides are currently not displayed,
+                // because the values for Windows are read from the main
+                // EpicOnlineServicesConfig.json file.
+                if (platform == PlatformManager.Platform.Windows)
+                {
                     continue;
+                }
 
-                toolbarStrings.Add(editor.GetLabelText());
+                if (!PlatformManager.TryGetConfigType(platform, out Type configType) || null == configType)
```

Comment:
It does _feel_ weird, but the circumstance is a valid one. It's possible for a platform to be defined and the config type to not be defined - in which case it would return true indicating that the key `platform` exists in the dictionary, but that the value _at_ that key is `null`. 

This will be the case (for instance) for the key with the value `PlatformManager.Platform.Unknown`.

Reply:
That's definitely... strange. I'll take your word for it that this is how it needs to be, though.